MSAL/test/app/mac/MSALAcquireTokenViewController.m

```diff
@@ -467,4 +524,41 @@ - (MSALAccount *)selectedAccount
     return self.accounts[self.userPopup.indexOfSelectedItem-1];
 }
 
+- (MSALInteractiveTokenParameters *)tokenParamsWithSSOSeeding:(BOOL)isSSOSeedingCall
+{
+    MSALTestAppSettings *settings = [MSALTestAppSettings settings];
+    MSALInteractiveTokenParameters *parameters = [[MSALInteractiveTokenParameters alloc] initWithScopes:isSSOSeedingCall ? [MSALTestAppSettings getSSOSeedingScope] : [settings.scopes allObjects]
+                                                                                      webviewParameters:[self msalTestWebViewParameters]];
+    
+    if (isSSOSeedingCall)
+    {
+        parameters.authenticationScheme = [MSALAuthenticationSchemeBearer new];
+    }
+    else
+    {
+        parameters.authenticationScheme = [self authScheme];
+    }
+    
+    parameters.loginHint = [self.loginHintTextField stringValue].length ? [self.loginHintTextField stringValue] : nil;
```

Comment:
nit: can we just do parameters.loginHint = self.loginHintTextField stringValue?

Reply:
Sure, I will update in next commit